labels.json

```diff
@@ -1,135 +1,316 @@
 [
   {
-    "name": "backwards-incompatible",
-    "color": "eb6420",
+    "name": "support",
+    "description": "Issue with a support-like question",
+    "color": "ebefc9",
     "aliases": [
-      "backwards-incompatible",
-      "breaking"
+      "support",
+      "question"
     ]
   },
   {
-    "name": "beginner-friendly",
-    "color": "d4c5f9",
+    "name": "new functionality request",
+    "description": "Issue with a request to add functionality",
+    "color": "006b75",
     "aliases": [
-      "beginner",
-      "beginner friendly"
+      "new functionality request",
+      "enhancement",
+      "feature"
+    ]
+  },
+  {
+    "name": "bug report",
+    "description": "Issue raised reporting a potential bug",
+    "color": "f7c6c7",
+    "aliases": [
+      "bug report",
+      "bug"
     ]
   },
   {
-    "name": "bug",
+    "name": "bug fix",
+    "description": "Pull Request raised fixing a potential bug",
     "color": "0e8a16",
     "aliases": [
-      "bug",
+      "bug fix",
       "bugfix"
     ]
   },
   {
     "name": "docs",
-    "color": "006b75",
+    "description": "Any Issue/Pull Request that related to documentation",
+    "color": "fef2c0",
     "aliases": [
       "docs",
       "documentation"
     ]
   },
   {
-    "name": "duplicate",
-    "color": "cccccc",
+    "name": "technical-debt",
+    "description": "Any Issue that raises changes that should be done, and any Pull Request that does them",
```

Comment:
«Any Issue that raises changes that should be done», that is the definition of an issue.